packages/engine-server/package.json

```diff
@@ -40,6 +40,7 @@
     "@dendronhq/common-all": "^0.69.1",
     "@dendronhq/common-server": "^0.69.1",
     "@dendronhq/remark-mermaid": "^0.2.0",
+    "@jcoreio/async-throttle": "^1.4.3",
```

Comment:
Can you make a corresponding change to `package.json` of plugin-core to remove the cli dependency and make sure the build still works?

https://github.com/dendronhq/dendron/blob/1ae79c528d15b3a77a13362b1170dfe064913391/packages/plugin-core/package.json#L903

Reply:
yep, can confirm. go ahead and delete